pkg/entities/offchain_tip_bot.go

```diff
@@ -94,7 +94,7 @@ func (e *Entity) TransferToken(req request.OffchainTransferRequest) ([]response.
 	insufficientBalanceLog := log
 	insufficientBalanceLog.Status = consts.OffchainTipBotTrasferStatusFail
 	insufficientBalanceLog.FailReason = consts.OffchainTipBotFailReasonNotEnoughBalance
-	senderBalance, err := e.svc.MochiPay.GetBalance(senderProfile.ID, req.Token)
+	senderBalance, err := e.svc.MochiPay.GetBalance(senderProfile.ID, req.Token, "0")
```

Comment:
temp keep this to avoid crash server. When upadte airdrop / tip need change this